docs/tutorials/github-keys.md

```diff
@@ -6,25 +6,24 @@ sidebar_label: GitHub Keys
 
 ## Why we need GitHub Keys?
 
-Let's first define GitHub as a browser-based online control system of distributed version for those software developers using the Git revision control system. Git uses SSH public key authentication and not a password authentication.  Each user of the system has to generate a key. 
+Let's first define GitHub as a browser-based online control system of distributed version for those software developers using the Git revision control system. Git uses SSH public key authentication and not a password authentication.  Each user of the system has to generate a key.
 
 The `SSH` is also known as Secure Shell or Secure Socket Shell [More About SSH in GitHub](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/about-ssh). SSH keys are used to authenticate secure connections they are generate through a public key cryptographic algorithm, the most common being RSA or DSA. This is a one-way formula that ensures the public key can be derived from the private key but the private key cannot be derived from the public key.
 
-
 ## How to generate a new SSH key?
```

Comment:
here too

Reply:
mm not sure about this one, talking with Xavier, we agree that exist two cases to resolves this, first one is to remove the question mark and capitalize the sentences or well don't capitalize and use the question mark which what i did here